src/classes/RequestHandler.py

```diff
@@ -30,64 +38,72 @@ def get_table_connection(self):
     def insert_resource(self, generated_uuid, current_time, resource):
         ddb_response = self.table.put_item(
             Item={
-                'resource_identifier': generated_uuid,
-                'modifiedDate': current_time,
-                'createdDate': current_time,
-                'metadata': resource['metadata']
+                Constants.DDB_FIELD_RESOURCE_IDENTIFIER: generated_uuid,
+                Constants.DDB_FIELD_MODIFIED_DATE: current_time,
+                Constants.DDB_FIELD_CREATED_DATE: current_time,
+                Constants.DDB_FIELD_METADATA: resource.metadata,
+                Constants.DDB_FIELD_FILES: resource.files,
+                Constants.DDB_FIELD_OWNER: resource.owner
             }
```

Comment:
what about status and other publishing data? Is'nt there a default value for that?

Reply:
Good point. We have discussed this, and such functionality must and will be implemented at a later time!